### ISSUE TITLE
Bugs with Damage Calculation for Mortal Wounds

### ISSUE THREAD

**Issue by Likewise42** [damonhook/aos-statshammer#71]
Hello! I was messing around with stats hammer and noticed some inconsistencies with averages.

These inputs give me outputs that degrade when you increase the save which is not right.
![image](https://user-images.githubusercontent.com/10952416/127705885-750047f6-9114-4fe8-b007-797a30ad9e9b.png)
![image](https://user-images.githubusercontent.com/10952416/127706070-3be3f30f-f9f2-402a-aaeb-351ccc5119f3.png)

And these inputs are mathematically equivalent, yet result in different outputs.
![image](https://user-images.githubusercontent.com/10952416/127705943-975ffe84-9a87-4c99-9425-e48e3e21dfce.png)

On chrome 92.0.4515.107/Windows if that helps!

**Comment by sedulam** [comment]
I'm getting the same behavior. I'm trying to replicate the Huskard on Thunderthusk's Blood Vulture:
![image](https://user-images.githubusercontent.com/9039639/128164133-a7946194-e732-4323-baf0-4c1d83d18578.png)

**Comment by damonhook** [comment]
So what I believe is happening is the follows:
After calculating the number of hits resulted in mortal wounds, it then has to deduct that from the number of normal (non MW) hits (since it is not "in addition").

Where this becomes an issue is when the chance to get a MW is higher than the base hit chance (thus more MW hits than normal ones). So when the deduction happens, it lands up doing negative hits.

TLDR: It should be an easy enough fix, just need to put a lower bound to the deductions.

**Comment by damonhook** [comment]
I have rolled out a fix in version 2.1.5

![Screenshot 2021-08-11 at 15 05 27](https://user-images.githubusercontent.com/35456894/129033861-7d4c1b8a-0a51-45ed-bc0f-89738cb70778.png)
